From a84bbb649db260910513d4ad9d07a4e41f166d99 Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Tue, 15 Jul 2014 16:55:10 -0500
Subject: [PATCH 3/8] TST: ARM: dts: OMAP4/5: Add iommu test device nodes

Add a iommu test device node on OMAP4 and OMAP5 for testing.
This device node will probe a test driver which allows us to
perform a iommu unit test.

This patch enables the testing of DSP IOMMU devices.

Signed-off-by: Suman Anna <s-anna@ti.com>
---
 arch/arm/boot/dts/omap4.dtsi | 6 ++++++
 arch/arm/boot/dts/omap5.dtsi | 6 ++++++
 2 files changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/omap4.dtsi b/arch/arm/boot/dts/omap4.dtsi
index 1a96d4317c97..c4efb9c65bfd 100644
--- a/arch/arm/boot/dts/omap4.dtsi
+++ b/arch/arm/boot/dts/omap4.dtsi
@@ -626,6 +626,12 @@
 			};
 		};
 	};
+
+	iommu_test {
+		compatible = "ti,omap-iommu-test";
+		iommus = <&mmu_dsp>;
+		/*iommus = <&mmu_ipu>;*/
+	};
 };
 
 #include "omap4-l4.dtsi"
diff --git a/arch/arm/boot/dts/omap5.dtsi b/arch/arm/boot/dts/omap5.dtsi
index 574ac11c0489..d5fcf08acdfd 100644
--- a/arch/arm/boot/dts/omap5.dtsi
+++ b/arch/arm/boot/dts/omap5.dtsi
@@ -1177,6 +1177,12 @@
 			>;
 		};
 	};
+
+	iommu_test {
+		compatible = "ti,omap-iommu-test";
+		iommus = <&mmu_dsp>;
+		/*iommus = <&mmu_ipu>;*/
+	};
 };
 
 &cpu_thermal {
-- 
2.22.0

